core-feature-pack/common/src/main/resources/modules/system/layers/base/org/slf4j/main/module.xml

```diff
@@ -28,6 +28,6 @@
     </resources>
 
     <dependencies>
-        <module name="org.slf4j.impl"/>
+        <module name="org.slf4j.impl" services="import"/>
```

Comment:
Why?

Reply:
Previously SLF4J used static binding and you had to name the implementation something like `org.slf4j.StaticLoggerBinder`. In SLF4J 2.0 they switched to using a `ServiceLoader` which is why the services need to be imported from the implementation.